packages/itinerary-body/src/ItineraryBody/index.tsx

```diff
@@ -30,6 +30,7 @@ const ItineraryBody = ({
   setActiveLeg,
   setLegDiagram,
   setViewedTrip,
+  shouldAlwaysCollapseAlerts = false,
```

Comment:
How about renaming this prop to simply `alwaysCollapseAlerts`? (re-sort props if you decide to rename)

Reply:
Good call, updated everywhere. Thanks.